code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1108,3 +1108,10 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	message_admins("<span class='notice'>[key_name(usr)] has equipped an loadout of type [outfit_type] to [key_name(M)].</span>", 1)
 
 	feedback_add_details("admin_verb","ELO") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+
+/proc/select_loadout()
+	var/list/outfits = (typesof(/datum/outfit/) - /datum/outfit/ - /datum/outfit/striketeam/)
```

Comment:
You can use `subtypesof()` instead of manually removing `/datum/outfit`. Unfortunately you cannot use `existing_typesof()` to also remove the other abstract type[s] because these are datums, but oh well.

Reply:
ah, okay.